tests/test_api.py

```diff
@@ -1840,18 +1851,21 @@ def __init__(self, platform, release, platforms, arrangement_version,
             'scratch': None,
             'platforms': platforms,
             'release': release,
-            'git_uri': None,
-            'git_ref': None,
-            'git_branch': None,
+            'git_uri': TEST_GIT_URI,
+            'git_ref': TEST_GIT_REF,
+            'git_branch': TEST_GIT_BRANCH,
             'user': None,
             'tag': None,
             'target': None,
             'architecture': None,
             'yum_repourls': None,
         }
-        if worker:
+        if arrangement_version:
             expected_kwargs.update({
                 'arrangement_version': arrangement_version,
+            })
+        if koji_upload_dir:
```

Comment:
on line 1829: koji_upload_dir = 'upload' if worker else None
koji_upload_dir is defined if worker

condition before was if worker, but that changed, so in order not to pass None koji_upload_dir, there is new condition for it